src/ASEconvert.jl

```diff
@@ -147,4 +147,7 @@ end
 # TODO Could have a convert_ase(Vector{Vector{Unitful}}) to make an ASE cell
 # TODO Could have a way to make an ASE calculator from an InteratomicPotential object
 
+include("atoms_calculators.jl")
```

Comment:
Ok, then maybe put the other stuff into a file as well

Reply:
I moved the conversions to own file. The possible issue here is where to have exports and imports. I am not completely sure is the way I split it good way to do it. But feel free to comment.

Reply:
For larger projects I'm not sure myself. I think for small packages like this, it's better to have the exports in one place (i.e. in the main package file).